### PR TITLE
Added version 1.0 and release date

### PR DESCRIPTION
**Pull request by bergos**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
   <script src="https://www.w3.org/Tools/respec/respec-w3c-common" class="remove"></script>
   <script class="remove">
     var respecConfig = {
-      specStatus: "CG-DRAFT",
+      specStatus: "CG-FINAL",
+      publishDate: "2019-03-27",
       shortName: "rdfjs-dataset",
-      subtitle: "RDF/JS: Dataset specification",
+      subtitle: "RDF/JS: Dataset specification 1.0",
       processVersion: 2018,
```
Comment (vhf) [inline index.html]: I would remove `processVersion`, it's been [removed](https://github.com/w3c/respec/blob/develop/CHANGELOG.md#v1950-2018-02-15).

**Comment by BigBlueHat** [comment]
Pretty version: https://raw.githack.com/rdfjs/dataset-spec/fc7916370d84c0b5b0ac150898f8e56fb641c453/index.html

**Comment by BigBlueHat** [comment]
Would it make sense to have a change log at this point? Or is it still too early? The [JSON-LD 1.1 Syntax spec](https://w3c.github.io/json-ld-syntax/#changes-from-10) has on showing changes from 1.0. Not sure if showing changes from "pre-1.0" makes sense or not--given that this wasn't formally version previously (afaik). Would be great to see a summary of new things/changes, though.

Thanks for doing this work, @bergos et al!

**Comment by vhf** [comment]
@BigBlueHat I'm not sure it makes sense because the DatasetCore interface is brand new. It is a good remark though, it might be a good idea to include some 'history' later on when we finalize the Dataset spec, highlighting the differences between it and the previous 'version' available here: https://github.com/rdfjs/data-model-spec/wiki/Dataset-spec/5c649e53dfc7db0b127cafdf051dca01b5c4c675

---

Unrelated to the above: ping @rubensworks @blake-regalia and @elf-pavlik, who made big contributions to this process but haven't been pinged yet or haven't reviewed yet, to make sure you see this PR. :)

**Comment by bergos** [comment]
The [version on the wiki](https://github.com/rdfjs/data-model-spec/wiki/Dataset-spec/5c649e53dfc7db0b127cafdf051dca01b5c4c675) describes the RDF-Ext Dataset API. The discussion about it lead to this spec, but it's not like a previous version. So I think a changelog to the that version doesn't make sense, but we should have it from 1.0 on.

**Pull request merged**